src/useragents.js

```diff
@@ -936,8 +936,9 @@ module.exports.useragents = {
       , mobile: true
       , a: true
       }
-    , 'Mozilla/5.0 (Mobile; Windows Phone 8.1; Android 4.0; ARM; Trident/7.0; Touch; rv:11.0; IEMobile/11.0; Microsoft; Lumia 535)': {
+    , 'Mozilla/5.0 (Mobile; Windows Phone 8.1; Android 4.0; ARM; Trident/7.0; Touch; rv:11.0; IEMobile/11.0; Microsoft; Lumia 640 LTE) like iPhone OS 7_0_3 Mac OS X AppleWebKit/537 (KHTML, like Gecko) Mobile Safari/537': {
```

Comment:
Yes this one is a real one. The previous one was not complete, that's why I edit it instead of creating a new one.